docs/tutorials/plots.rst

```diff
@@ -1057,3 +1059,39 @@ make multiple plots:
     plt.show()
 
 :ref:`Return to Top <plots>`
+
+.. _finder_image:
+
+Finder Chart/Image
+==================
+
+`astroplan` includes a function for generating quick finder images from the
+command line, `~astroplan.plots.plot_finder_image`, by querying for images from
+sky surveys centered on a `~astroplan.FixedTarget`. This function depends on
+`astroquery <https://astroquery.readthedocs.org/en/latest/>`_  (in addition
+to `Matplotlib`_). In this example, we'll quickly make a finder image centered
+on The Crab Nebula (M1):
+
+.. code-block:: python
```

Comment:
I didn't know this command, and it seems not to be working. Should that be placed like so: 
```

.. plot::
   :include-source:
    from astroplan import FixedTarget
    from astroplan.plots import plot_finder_image
    import matplotlib.pyplot as plt

    messier1 = FixedTarget.from_name("M1")
    ax, hdu = plot_finder_image(messier1)
    plt.show()
```